gossipd/gossipd.c

```diff
@@ -380,8 +380,9 @@ static void handle_discovered_ip(struct daemon *daemon, const u8 *msg)
 	return;
 
 update_node_annoucement:
-	status_debug("Update our node_announcement for discovered address: %s",
-		     fmt_wireaddr(tmpctx, &discovered_ip));
+	if (daemon->ip_discovery)
```

Comment:
Is this test correct?

Reply:
Nope, it was not very precise. Fixed that